docker-compose.override.yml

```diff
@@ -5,7 +5,7 @@ services:
     build:
       context: .
       target: gulp
-    command: bash -c "npx gulp assemble && npx gulp watch"
+    command: bash -c "npx gulp assemble --environment=development --sass-lint=false && npx gulp watch --environment=development --sass-lint=false"
```

Comment:
The default should be to have sass-lint on.

Reply:
Will make it an option, but I find it pretty annoying during development.